### ISSUE TITLE
Report test failures due to signals

### ISSUE THREAD

**Issue by sfackler** [nextest-rs/nextest#281]
If a test fails due to a signal (e.g. a SIGSEGV), nextest currently just reports it as a generic failure, probably with no stdout:

```
        FAIL [   0.010s]              openssl pkey::tests::test_ec_gen

--- STDOUT:                           openssl pkey::tests::test_ec_gen ---

running 1 test
```

It would be great if the runner looked at the process's exit code to see if it happened due to a signal and expose that to provide more context.

**Comment by sunshowers** [comment]
Thanks! Yeah, I agree that this could be better. We do capture stdout though no matter what case it is.

**Comment by sunshowers** [comment]
This has been released as part of cargo-nextest 0.9.21 (binaries should be available in the next few minutes). Thanks again for the report!
